src/reducers/eventSchedules-list.js

```diff
@@ -4,12 +4,34 @@ import {
   RESET_EVENTS_SCHEDULE,
 } from "../actions/eventSchedule/eventSchedule-list-action";
 
+const parseReccurentEvents = data =>
+  data.items.map(
+    ({
+      eventId: id,
+      frequency,
+      isActive,
+      lastRun,
+      nextRun,
+      periodicity,
+      title,
+    }) => ({
+      id,
+      frequency,
+      isActive,
+      lastRun,
+      nextRun,
+      periodicity,
+      title,
+    }),
+  );
+
```

Comment:
create new file for parser for this reducer

Reply:
fixed